docker.go

```diff
@@ -309,7 +309,10 @@ func commandPush(build Build, tag string) *exec.Cmd {
 
 // helper function to create the docker daemon command.
 func commandDaemon(daemon Daemon) *exec.Cmd {
-	args := []string{"--data-root", daemon.StoragePath}
+	args := []string{
+		"--data-root", daemon.StoragePath,
+		"--host=unix:///var/run/docker.sock",
```

Comment:
this would break the plugin for windows.  Are we sure this is necessary since `DOCKER_HOST` is being set as an image environment variable?

Reply:
The other solution would be to listen on TCP socket without TLS, like this `tcp://0.0.0.0:2375`.
Should I make the change ?